src/app/auth/CertErrorComponent.tsx

```diff
@@ -14,14 +14,14 @@ class CertErrorComponent extends React.Component<any> {
       <div>
         {this.props.certError ? (
           <div>
-            A certificate error has occurred, likely due to the usage of self signed certificates in
-            one of the clusters. Please try to visit the failed url and accept the CA:
+            A certificate error has occurred, likely caused by using self-signed CA certificates in
+            one of the clusters. Navigate to the following URL and accept the certificate:
             <a href={this.props.certError.failedUrl}> {this.props.certError.failedUrl}</a>
             <div />
-            The correct way to address this is to install your self CA into your browser.
+            If an "Unauthorized" message appears after you have accepted the certificate, refresh
+            the web page.
```

Comment:
@eriknelson 
I just want to confirm something. Does "refresh the web page" mean that the user should reload the OAuth server URL or the MTC console?